pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: ordered_set
 description: A simple implementation of an Ordered Set for Dart that allows multiple items with the same priority.
-version: 5.0.3
-homepage: https://github.com/luanpotter/ordered_set
+version: 6.0.0
+homepage: https://github.com/bluefireteam/ordered_set
```

Comment:
Drive by fix. Unless keeping the homepage as https://github.com/luanpotter/ordered_set was intentional, in which case it's easy to remove this change.